Bio/Align/__init__.py

```diff
@@ -16,12 +16,10 @@
 
 import warnings
 
-from Bio import BiopythonDeprecationWarning
+from Bio import Alphabet, BiopythonDeprecationWarning
```

Comment:
Personally I would keep this as two lines, but like with the resorting of the imports.

Reply:
This is automatically done by my editor. The utility responsible for this is isort. Maybe after you decided to keep you code style consistent with black, you might want to think about keeping your imports consistent with isort. 

Anyway, I will manually separate the two again.

Reply:
This is automatically done by my editor. The utility responsible for this is isort. Maybe after you decided to keep you code style consistent with black, you might want to think about keeping your imports consistent with isort. 

Anyway, I will manual separate the two again.